example/views.py

```diff
@@ -74,8 +74,13 @@ class AuthorViewSet(ModelViewSet):
 
 
 class CommentViewSet(ModelViewSet):
-    queryset = Comment.objects.all()
+    queryset = Comment.objects.select_related('author', 'entry')
```

Comment:
So, it really happens in https://github.com/ABASystems/django-rest-framework-json-api/commit/b96d79d49995c0faac6174ecdd1715a2b1121d3d merged a year ago.

@jerel do you remember the conversation that was around this ?